unstructured/metrics/text_extraction.py

```diff
@@ -50,3 +52,49 @@ def calculate_edit_distance(
     elif return_as == "distance":
         return distance
     return 0.0
+
+
+# Duplicate code from cleaners.core,
```

Comment:
Could we call the functions in cleaners.core instead of duplicating?

Reply:
The function in cleaners.core doesn't allow you to exclude punctuation, and I didn't know if we wanted to introduce that functionality to that function.

Reply:
Ah ok, if it's doing something different then it's fine to just duplicate/adapt it here without a comment. if it's better to centralize these types of cleaning functions to one file, perhaps we should create a new function like `remove_sentence_punctuation` (or something) in cleaners.core and call it here, so it can be used for other purposes other than metrics as well.